compiler/optimizer/OMRValuePropagation.hpp

```diff
@@ -281,118 +284,32 @@ class ValuePropagation : public TR::Optimization
    ValueConstraint *copyValueConstraints(ValueConstraints &valueConstraints);
 
 #if USE_TREES
-   class ValueConstraintHandler : public TREE_HANDLER<ValueConstraint>
+   class ValueConstraintHandler : public TREE_HANDLER <ValueConstraint>
       {
       public:
-      void setVP(TR::ValuePropagation *vp) {_vp = vp;}
```

Comment:
Sorry, I'm going to bow out of this one, mostly because I don't think I have enough background on what's being asked here. If there's still confusion Monday, maybe Andrew and Liqun and I can talk about this in person with a whiteboard, and get the background filled in. 

Hopefully it doesn't come to that tho, and @xliang6 and @0xdaryl have answers :)